assignments-list/snippet.html

```diff
@@ -0,0 +1,128 @@
+<style>
```

Comment:
This file does not match the one described in the README.

It appears to be a duplicate of the Filedrop slideshow